### ISSUE TITLE
Dovecot's managesieve port is not opened by default

### ISSUE THREAD

**Issue by stevenroose** [YunoHost/moulinette-yunohost#78]
The YunoHost Dovecot installation supports Sieve and managesieve. However, the managesieve port (4190) is not opened in the firewall by default.

Is this on purpose? Security consideration perhaps? Or just forgotten?
Roundcube has the managesieve installation configured by default, but uses the local interface, so it bypasses the firewall.

**Comment by abeudin** [comment]
Hi,  can you give me example for why you want access at this port from external IP?

**Comment by opi** [comment]
@abeudin I think it's for managing sieve filter from a desktop client

**Comment by stevenroose** [comment]
Yes, Thunderbird and KMail f.e. have a managesieve interface to configure
Sieve. It is also the only way to do it, if Roundcube is not installed...

**Comment by JocelynDelalande** [comment]
Quite new to ynh code, but wouldn't it rather be a bug on https://github.com/YunoHost/yunohost-config-dovecot/ ?

**Comment by julienmalik** [comment]
@JocelynDelalande 

the bug is at least here : https://github.com/YunoHost/moulinette-yunohost/blob/unstable/data/other/firewall.yml

the managesieve daemon should bind to all the interfaces IP too, but if @stevenroose tested it by only opening the port in the firewall, I guess it is safe to open the port by default in Yunohost by just updating firewall.yml

**Comment by JocelynDelalande** [comment]
@julienmalik ok, thanks for taking time to explain :)

**Comment by Kloadut** [comment]
+1 to open the port by default
